nc_time_axis/tests/integration/test_plot.py

```diff
@@ -0,0 +1,37 @@
+"""Integration test for plotting data with non-gregorian calendar."""
+
+from __future__ import (absolute_import, division, print_function)
+from six.moves import (filter, input, map, range, zip)  # noqa
+
+import unittest
+
+import netcdftime
+import matplotlib
+matplotlib.use('agg')
+import matplotlib.pyplot as plt
+
+import nc_time_axis
+
+
+class Test(unittest.TestCase):
+    def setUp(self):
+        # Make sure we have no unclosed plots from previous tests before
+        # generating this one.
+        plt.close('all')
+
+    def tearDown(self):
+        # If a plotting test bombs out it can leave the current figure
+        # in an odd state, so we make sure it's been disposed of.
+        plt.close('all')
+
+    def Test_360_day_calendar(self):
+        dates = [nc_time_axis.CalendarDateTime(netcdftime.datetime(
+            1986, month, 30), '360_day') for month in range(1, 6)]
+        line1, = plt.plot(dates)
+        result_ydata = line1.get_ydata()
+        self.assertArrayEqual(expected_ydata, result_ydata)
```

Comment:
@lbdreyer Can you test this in your conda environment with the appropriate `netcdf` version.

Also make sure `python setup.py test` is discovering and running this integration test ...